dev/com.ibm.ws.microprofile.faulttolerance.2.0.cdi/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_2_0.xsd"
+    version="2.0" bean-discovery-mode="all">
+
+    <scan>
+        <exclude
+            name="com.ibm.ws.microprofile.faulttolerance.cdi20.*" />
+    </scan>
```

Comment:
It seems odd to be setting `bean-discovery-mode=all` and then excluding all the classes within it from scanning. Should this not be `bean-discovery-mode=none`?

Might even be able to remove it since there's an extension in there.

I hate our CDI extension mechanisms :disappointed:

Reply:
There are a great many bundles that do it this way. I didn't fancy changing all of those today. So I opted for consistency with FT 1.0